data/formats-data.js

```diff
@@ -4022,7 +4029,7 @@ exports.BattleFormatsData = {
 		tier: "RU",
 	},
 	croagunk: {
-		randomBattleMoves: ["fakeout", "vacuumwave", "suckerpunch", "drainpunch", "darkpulse", "knockoff", "gunkshot", "toxic"],
+		randomBattleMoves: ["bulkup", "nastyplot", "fakeout", "vacuumwave", "suckerpunch", "drainpunch", "focusblast", "knockoff", "gunkshot", "sludgebomb"],
```

Comment:
+ shadow ball for special set

Reply:
blast+bomb+(vacuum/knock/gunk/whatever) sounds adequate and the amount of scenarios where other move options will outweigh it just seems way too many